src/current/v23.2/cockroach-debug-merge-logs.md

```diff
@@ -39,44 +40,48 @@ Generate a debug zip file:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ cockroach debug zip ./cockroach-data/logs/debug.zip --insecure
+cockroach debug zip ./cockroach-data/logs/debug.zip --insecure
 ~~~
 
 Unzip the file:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ unzip ./cockroach-data/logs/debug.zip
+unzip ./cockroach-data/logs/debug.zip
 ~~~
 
 Merge the logs in the debug folder:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ cockroach debug merge-logs debug/nodes/*/logs/*
+cockroach debug merge-logs debug/nodes/*/logs/* --format=crdb-v1
 ~~~
 
 Alternatively, filter the merged logs for a specified time range:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-$ cockroach debug merge-logs debug/nodes/*/logs/* --from="220713 18:36:28.208553" --to="220713 18:36:29.232864"
+cockroach debug merge-logs debug/nodes/*/logs/* --format=crdb-v1 --from="231031 19:19:50.917185" --to="231031 19:19:57.189263"
 ~~~
 
 You can also filter the merged logs for a regular expression:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-cockroach debug merge-logs debug/nodes/*/logs/* --filter="RUNNING IN INSECURE MODE"
+cockroach debug merge-logs debug/nodes/*/logs/* --format=crdb-v1 --filter="ALL SECURITY CONTROLS HAVE BEEN DISABLED"
 ~~~
 
 You can redact sensitive information from the merged logs:
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-cockroach debug merge-logs --redact debug/nodes/*/logs/*
+cockroach debug merge-logs debug/nodes/*/logs/* --format=crdb-v1 --redact
 ~~~
 
+## Consideration
+
+As of v23.2, logs can be configured to use a [timezone with formats `crdb-v1` or `crdb-v2`]({% link {{ page.version.version }}/configure-logs.md %}#set-timezone). This is a [backward-incompatible change](https://www.cockroachlabs.com/docs/releases/v23.2#v23-2-0-alpha-1-backward-incompatible-changes). With versions prior to v23.2, `cockroach debug merge-log` will return an error if run against logs with the timezone configured. `cockroach debug merge-log` requires v23.2 or later if run against logs with the timezone configured.
```

Comment:
Small typo `cockroach debug merge-log` -> `cockroach debug merge-logs`. I think the same typo was in the release-note (oops!)

Reply:
corrected typo. Published release note was correct.